counterparty-core/counterpartycore/test/regtest/testscenarios.py

```diff
@@ -113,6 +154,11 @@ def run_scenarios():
 
         for item in SCENARIOS:
             context = run_item(regtest_node_thread.node, item, context)
+
+        # print("Server ready, ctrl-c to stop.")
```

Comment:
remove?

Reply:
I used it often! I need to add some arguments to the script `testsenarios.py`